src/Agent.Service/Windows/Program.cs

```diff
@@ -12,33 +14,37 @@ static class Program
         /// </summary>
         static int Main(String[] args)
         {
-            if (args != null && args.Length == 1 && args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
+            if (args != null && args.Length >= 1)
```

Comment:
If we are asserting that an outer process is necessary for auto-logon agent, then why should interactively running the agent from the command prompt be less robust? Shouldn't run.cmd be modified to enjoy the same robustness?

Furthermore, why shouldn't running the agent interactively on Mac/Linux enjoy the same robustness?

Hacking the Windows Service .exe to be the bootstrapper process for auto-logon is the wrong place. It's a short sighted gain that only benefits Windows when running the auto-logon agent.